packages/dev/core/src/Behaviors/Meshes/pointerDragBehavior.ts

```diff
@@ -30,10 +30,15 @@ export class PointerDragBehavior implements Behavior<AbstractMesh> {
     private _beforeRenderObserver: Nullable<Observer<Scene>>;
     private static _PlaneScene: Scene;
     private _useAlternatePickedPointAboveMaxDragAngleDragSpeed = -1.1;
+    private _activeDragButton: number = -1;
     /**
      * The maximum tolerated angle between the drag plane and dragging pointer rays to trigger pointer events. Set to 0 to allow any angle (default: 0)
      */
     public maxDragAngle = 0;
+    /**
+     * Butttons that can be used to initiate a drag
+     */
+    public dragButtons = [0, 1, 2];

```

Comment:
shouldn't we limit this per default (while allowing other buttons to be added?) or will it be a breaking change?

Reply:
I am fine with whatever you decide :-) You can resolve this whenever you want.